ablySpec/RealtimeClientChannel.swift

```diff
@@ -552,6 +552,60 @@ class RealtimeClientChannel: QuickSpec {
 
                 }
 
+                // RTL6g
+                context("Identified clients with clientId") {
+
+                    // RTL6g1
+                    context("When publishing a Message with clientId set to null") {
+
+                        // RTL6g1a
+                        pending("should be unnecessary to set clientId of the Message before publishing") {
+                            let options = AblyTests.commonAppSetup()
+                            options.autoConnect = false
+                            let client = ARTRealtime(options: options)
```

Comment:
This is still wrong as far as I can see. This is supposed to be testing publishing a message without an explicit `clientId`, yet the client must be identified i.e. with a `clientId`.  This client is not identified